src/test/java/stepDefinitions/chpldownloadSteps.java

```diff
@@ -1,146 +1,145 @@
-	    }
-		
-	}
-	
-	@Given("^user is logged in with \"([^\"]*)\" and \"([^\"]*)\"$")
-	public void user_is_logged_in_with_and(String arg1, String arg2) throws Throwable {
-	    
-		driver.get("https://chpl.ahrqdev.org/#/resources/download");
-		driver.manage().window().maximize();
-		driver.manage().timeouts().implicitlyWait(60, TimeUnit.SECONDS);
-		driver.findElement(By.xpath("//*[@id='login-toggle']")).click();
-		Thread.sleep(100);
-		// This part is currently not logging in user; login button click() is not working
-		driver.findElement(By.id("username")).sendKeys("username");
-		driver.findElement(By.id("password")).sendKeys("password");
-		driver.findElement(By.xpath("//*[@id='admin']/li/div/form/button[1]")).click();
-		  
-	}
-
-	@When("^User checks download file box$")
-	public void user_checks_download_file_box() throws Throwable {
-		
-}
-	
-	@Then("^user sees 8 download files")
-	public void user_sees_8_download_files() throws Throwable {
-		
-		WebElement element=driver.findElement(By.name("downloadSelect"));
-		Select select=new Select(element);
-		List<WebElement> list=select.getOptions();
-		System.out.println("Number of download files: " +list.size());
-		for(int i=0;i<list.size();i++)          
-		    {
-		        System.out.println(list.get(i).getText());
-		    }
-	
-	}
-
-	
-	
-}
+package stepDefinitions;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import pageObjects.chpldownload_page;
+public class chpldownloadSteps {
+
+	public WebDriver driver;
+	
+	public chpldownloadSteps()
+	{
+		driver = hooks.driver;
+		
+	}
+	
+	@Given("^User is on CHPL home$")
+	public void user_is_on_CHPL_home() throws Throwable {
+		driver.get("https://chpl.ahrqdev.org/#/search");
+		WebDriverWait wait = new WebDriverWait(driver, 20);
+		chpldownload_page.print_page_title(driver);
+	}
+	
+	@When ("^User clicks on CHPL Resources button$")
+	public void user_clicks_on_CHPL_Resources_button() throws Throwable {
+		
+		//driver.findElement(By.xpath("//*[@id='resource-toggle']")).click();
+		//driver.findElement(By.xpath("//*[@id='resource-dropdown-menu']/li[4]/a")).click();
+		driver.get("https://chpl.ahrqdev.org/#/resources/download");
+		WebDriverWait wait = new WebDriverWait(driver, 20);
+		
+	}
+	
+	@Then("^Download the CHPL page loads successfully and page title shows correct$")
+	public void download_the_CHPL_page_loads_successfully_and_page_title_shows_correct() throws Throwable {
+	    
+		chpldownload_page.print_page_title(driver);
+	}
+
+	@Given("^user is not logged in$")
+	public void user_is_not_logged_in() throws Throwable {
+	System.out.println("user is not logged in");

```

Comment:
This `@Given` will need to be updated once log in works; it should check to see if the user is logged in, and if it is, should figure out how to click the "Log Out" button. Probably doesn't need to be done now, but eventually